composer/callbacks/health_checker.py

```diff
@@ -69,6 +62,14 @@ def __init__(
         if not self.slack_webhook_url:
             self.slack_webhook_url = os.environ.get('SLACK_WEBHOOK_URL', None)
 
+        if self.slack_webhook_url:
+            # fail fast if missing import
+            try:
+                import slack_sdk
+                del slack_sdk
+            except ImportError as e:
+                raise MissingConditionalImportError('health_checker', 'slack_sdk', None) from e
```

Comment:
```suggestion
        if self.slack_webhook_url:
            try:
                import slack_sdk
                del slack_sdk
            except ImportError as e:
                raise MissingConditionalImportError('health_checker', 'slack_sdk', None) from e
```
nit: i don't think comment is necessary (tho feel free to leave)

Reply:
skipping this one